app/controllers/api/base_controller/parameters.rb

```diff
@@ -169,11 +171,13 @@ def sort_params(klass)
       end
 
       def sort_directive(attr, order, options)
-        sort_item = attr
-        sort_item = "LOWER(#{sort_item})" if options.map(&:downcase).include?("ignore_case")
-        sort_item << " ASC"  if order && order.downcase.start_with?("asc")
-        sort_item << " DESC" if order && order.downcase.start_with?("desc")
-        sort_item
+        if options.map(&:downcase).include?("ignore_case")
```

Comment:
@jntullo if this was green we must have insufficient test coverage here....can you add some to verify existing behavior? Thanks!